From a1cd94598eb0298e8b1d2b5c9626ee0b866a922f Mon Sep 17 00:00:00 2001
From: Ben Peled <bpeled@marvell.com>
Date: Thu, 31 Jan 2019 11:25:26 +0200
Subject: [PATCH 1485/1921] dts: marvell: add 2 eeprom properties to A7K DB
 device tree

Add on-board i2c EEPROMs U37 and U38

Signed-off-by: Ben Peled <bpeled@marvell.com>
Signed-off-by: Konstantin Porotchkin <kostap@marvell.com>
Change-Id: I81a239b8a3d0de022dd60dc6d8dd7c2c7756ea67
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/50243
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db.dts | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dts b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
index de2e1e8d0f56..be985ba54f3f 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dts
@@ -152,6 +152,20 @@
 		 * IO0_7:		IO1_7: SDIO_Vcntrl
 		 */
 	};
+
+	/* U38 */
+	eeprom0: eeprom@50 {
+		compatible = "atmel,24c64";
+		reg = <0x50>;
+		pagesize = <0x20>;
+	};
+
+	/* U37 */
+	eeprom1: eeprom@57 {
+		compatible = "atmel,24c64";
+		reg = <0x57>;
+		pagesize = <0x20>;
+	};
 };
 
 &cp0_nand_controller {
-- 
2.31.1

